File: src/models/TeamMember.ts
import uuid from 'react-native-uuid';

export interface TeamMember {
  id: string;
  name: string;
  email: string;
  role: string;
  createdAt: Date;
}

export const createTeamMember = (
  name: string,
  email: string,
  role: string
): TeamMember => ({
  id: uuid.v4().toString(),
  name,
  email,
  role,
  createdAt: new Date()
}); 

File: app/src/main/java/com/example/pms/data/models/TeamMember.kt
package com.example.pms.data.models

import java.util.Date
import java.util.UUID

data class TeamMember(
    val id: String = UUID.randomUUID().toString(),
    val name: String,
    val email: String,
    val phone: String = "",
    val role: String,
    val skills: List<String> = emptyList(),
    val createdAt: Date = Date()
)
